brigadier/templates/nav.html

```diff
@@ -2,24 +2,44 @@
-        <a class="navbar-brand" href="#">Brigadier</a>
+            <a class="navbar-brand" href="{% url "home:home" %}?nb=Brigadier">{% translate "Brigadier" %}</a>
         <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
                 aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
             <span class="navbar-toggler-icon"></span>
         </button>
         <div class="collapse navbar-collapse" id="navbarSupportedContent">
             <ul class="navbar-nav me-auto mb-2 mb-lg-0">
                 <li class="nav-item">
-                    <a class="nav-link active" aria-current="page" href="{% url "home:home" %}">{% translate "Home" %}</a>
+                    {% if request.GET.nb == "Home" %}
+                        <a class="nav-link active" aria-current="page"
+                           href="{% url "home:home" %}?nb=Home">{% translate "Home" %}</a>
+                    {% else %}
+                        <a class="nav-link" aria-current="page"
+                           href="{% url "home:home" %}?nb=Home">{% translate "Home" %}</a>
+                    {% endif %}
                 </li>
                 <li class="nav-item">
-                    <a class="nav-link" href="{% url "projects:list" %}">{% translate "Projects" %}</a>
+                    {% if request.GET.nb == "Projects" %}
+                        <a class="nav-link active" href="{% url "projects:list" %}?nb=Projects">{% translate "Projects" %}</a>
+                    {% else %}
+                        <a class="nav-link" href="{% url "projects:list" %}?nb=Projects">{% translate "Projects" %}</a>
+                    {% endif %}
                 </li>
-                 <li class="nav-item">
-                    <a class="nav-link" href="{% url "projects:task_list" %}">{% translate "Tasks" %}</a>
+                <li class="nav-item">
+                    {% if request.GET.nb == "Tasks" %}
+                        <a class="nav-link active" href="{% url "projects:task_list" %}?nb=Tasks">{% translate "Tasks" %}</a>
+                    {% else %}
+                        <a class="nav-link" href="{% url "projects:task_list" %}?nb=Tasks">{% translate "Tasks" %}</a>
+                    {% endif %}
                 </li>
-                 <li class="nav-item">
-                    <a class="nav-link" href="{% url "employees:list" %}">{% translate "Employees" %}</a>
+                <li class="nav-item">
+                    {% if request.GET.nb == "Employees" %}
+                        <a class="nav-link active"
+                           href="{% url "employees:list" %}?nb=Employees">{% translate "Employees" %}</a>
+                    {% else %}
+                        <a class="nav-link"
+                           href="{% url "employees:list" %}?nb=Employees">{% translate "Employees" %}</a>
+                    {% endif %}
```

Comment:
you should rollback this file from master branch
and write code from very beginning
but this time without trying to shoot in a foot
this solution is very complicated and hardly maintainable
avoid additional parameters
use only current request path
```
{% if request.path == url ... %}active{% endif %}
```